recipe/conda_build_config.yaml

```diff
@@ -533,6 +533,8 @@ mkl:
   - 2022
 mkl_devel:
   - 2022
+mpg123:
+  - '1.31'
```

Comment:
I only see https://github.com/conda-forge/yaafe-feedstock/pulls that needs mpg123, and they haven't rebuilt in a while.

Ok as is.